### ISSUE TITLE
CQL: Default modifiers for WILDCARD, SINGLECHAR, and ESCAPECHAR are invalid according to the BNF

### ISSUE THREAD

**Issue by JoelPerren** [opengeospatial/ogcapi-features#567]
The spec states that the default modifiers are:

```
WILDCARD '%'
SINGLECHAR '_'
ESCAPECHAR '\' 
```

However, the BNF definitions for each are:

```
["WILDCARD" quote alpha quote]
["SINGLECHAR" quote alpha quote]
["ESCAPECHAR" quote alpha quote]
```

The BNF definition for `alpha` does not include non-alphabetical characters such as `%`, `_`, or `\`.

Another BNF grouping should be made for symbols and added and the BNF updated if this behaviour is desired.

E.g.:
```
## This includes the following: ! " # $ % & ' ( ) * + , - . / : ; < = > ? @ [ \ ] ^ _ ` { | } ~
symbol = "\x0021".."\x002F" | "\x003A".."\x0040" | "\x005B".."\x0060" | "\x007B".."\x007E"
...
["WILDCARD" quote (alpha | symbol) quote]
["SINGLECHAR" quote (alpha | symbol) quote]
["ESCAPECHAR" quote (alpha | symbol) quote]
```

**Comment by cportele** [comment]
2021-05-10:  This will be resolved by #541.

**Comment by pvretano** [comment]
This is now obsolete.  PR #592  removes all these qualifiers.

**Comment by pvretano** [comment]
Resolved by PR #592.  See also issues #541 and #555.
